### PR TITLE
build: Update dependencies and dependabot

### PR DESCRIPTION
**Pull request by riscait**
## 🙌 What's Done

- Update dependencies
- Update dependabot schedules

## ✍️ What's Not Done

-

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -43,7 +46,7 @@ updates:
         patterns:
           - "*"
         update-types:
-        - "patch"
+          - "patch"
```
Comment (riscait) [inline .github/dependabot.yml:49]: Fix indents.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -21,6 +21,9 @@ updates:
     schedule:
       interval: "weekly"
     groups:
+      altfire-dependencies:
+        patterns:
+          - "altfire*"
```
Comment (riscait) [inline .github/dependabot.yml:26]: For `altfire_xxx` packages.

File: melos.yaml [hunk 3]
```diff
@@ -104,17 +109,17 @@ scripts:
 
   pod:
     run: melos run pod:ios --no-select && melos run pod:macos --no-select
-    description: Run all pod install.
+    description: Clean install the Pods
 
   pod:ios:
-    exec: cd ios && pod install
+    exec: cd ios && rm -f Podfile.lock && pod install --repo-update
```
Comment (riscait) [inline melos.yaml:115]: Failed if podfile.lock is out of date, so clean install.

**Review by k-nkmr — APPROVED** [review]
LGTM

**Pull request merged**